engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -239,6 +247,20 @@ final class SuggestionsHandler(
       context.become(initialized(name, clients))
   }
 
+  /**
+    * Transition the initialization process.
+    *
+    * @param state current initialization state
+    */
+  private def becomeInitialized(
```

Comment:
If the initialization may not succeed, the method shouldn't have a name `becomeInitialized`. The better name would be `tryInitialize`.